suites/nautilus/ansible/tier-1_deploy_dmcrypt.yaml

```diff
@@ -0,0 +1,71 @@
+#===============================================================================================
+# Tier-level: 1
+# Test-Suite: tier-1_deploy.yaml
+# Test-Case: Deployment and rollover tests
+#
+# Cluster Configuration:
+#    cephci/conf/nautilus/ansible/tier-1_deploy.yaml
+#
+#    7-Node cluster(RHEL-7.9 and above)
+#    3 MONS, 2 MDS, 1 MGR, 3 OSD and 2 RGW service daemon(s)
+#     Node1 - Mon, Mgr, Installer, OSD
+#     Node2 - Mon, Mgr, OSD
+#     Node3 - Mon, OSD
+#     Node4 - Client, Grafana
+#     Node5 - Pool , used for roll-over test case
+#     Node6 - Pool , used for roll-over test case
+#     Node7 - Pool , used for roll-over test case
+#
+# Test Steps:
+#   (1) Install Pre-requisites, and Deploy Ceph using ceph-ansible
+#   (2) Use pool nodes to add/remove MON, OSD, RGW and MDS using roll-over playbook.
+#===============================================================================================
+tests:
+   - test:
+      name: install ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+
+   - test:
+      name: ceph ansible
+      polarion-id: CEPH-10961,CEPH-83572954,CEPH-83572951
```

Comment:
It is recommended to have a single polarion id for direct mapping unless we have a mechanism to map failures to the right ID.

Reply:
@psathyan here actually these two test cases (CEPH-83572954,CEPH-83572951) are covered in same inventory should i still skip it or create new one for each?

Reply:
Having multiple test cases clubbed into a single script without the ability to differentiate which test case has failed is futile.

IMO, we should fix the test case in both places and ensure we are able to capture the right result. This helps when TFA is involved.

Reply:
@psathyan i have update the polarion tc

Reply:
Thank you.